src/tools/miri/tests/pass/function_calls/target_feature_wasm.rs

```diff
@@ -0,0 +1,11 @@
+//@only-target-wasm32: tests WASM-specific behavior
```

Comment:
Currently we wouldn't even run this test; [these lines](https://github.com/rust-lang/miri/blob/318c4211df1c8df5a30f8d8f3503133ff4f90dcc/ci.sh#L113-L114) control what we run on wasm. Can you add `wasm` there so that the filters pick up the new test you added?

Reply:
I manually ran the test, but I didn't notice it wouldn't run as part of the test suite.